src/app.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import Header from './header';
+
+import JudgingRoundDisplay from './judging_round_display';
+import JudgingRoundSelector from './judging_round_selector';
+
+
+class App extends React.Component {
+    constructor(props) {
+	super(props);
+	this.state = {
+            judging_round: null,
```

Comment:
It looks like you have `transform-class-properties` setup in your [babelrc](https://github.com/masschallenge/application-allocator/blob/development/.babelrc#L3) so you can remove this constructor call and have:

```
class App extends React.Component {
  state = { judging_round: null }
...
```

This will be the default syntax in future react versions so that babel setting can eventually be removed. It's a nice change since it removes a good bit of boilerplate :)

Reply:
Done.